src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/BaseConvertDataFunctionalTests.cs

```diff
@@ -350,5 +363,21 @@ protected void ConvertJsonMessageAndValidateExpectedResponse(ITemplateProvider t
 
             Assert.True(JToken.DeepEquals(expectedObject, actualObject));
         }
+
+        protected void ConvertFhirBundleToHl7v2AndValidateExpectedResponse(ITemplateProvider templateProvider, string rootTemplate, string inputFile, string expectedFile)
+        {
+            var jsonProcessor = new FhirToHl7v2Processor(_processorSettings, FhirConverterLogging.CreateLogger<FhirToHl7v2Processor>());
+            var inputContent = File.ReadAllText(inputFile);
+            var expectedContent = File.ReadAllText(expectedFile);
+            var actualContent = jsonProcessor.Convert(inputContent, rootTemplate, templateProvider);
+
+            var fieldValues = actualContent.Split("|");
+
+            // remove MSH-7 timestamp
+            fieldValues[6] = string.Empty;
```

Comment:
What is the timestamp here in this context? 
Asking cause with Rob's change to handle date time parsing, we should probably validate any timestamp fields are being preserved as is without UTC conversion.

Reply:
MSH-7 the time that the HL7v2 message was generated. I just used the current time for populating MSH-7 which will always be changing whenever the tests run so I excluded the timestamp from validation. The timestamp isn't really used for anything important. Some systems use it for searching for messages in certain time range but it does not have any clinical impact

Reply:
Got it, just wanted to ensure it wasn't supposed to be a timestamp value from the input FHIR bundle/resource.